### ISSUE TITLE
Will Fintel ever add Beneish m score and/or John Campbells probability of financial distress models to their stock screener?

### ISSUE THREAD

**Issue by HenryMoughalian** [risenhoover/fintel-issues#40]
Will Fintel ever add Beneish m score and/or John Campbell's probability of financial distress mode? If you guys don't know what that is explained in this business insider article here; https://www.businessinsider.com/the-beneish-m-score-identifying-earnings-manipulation-and-short-candidates-2011-5

If you guys don't know what John Campbell's probability of financial distress model it is also explained in this NASDAQ article here; https://www.nasdaq.com/articles/new-calculation-probability-financial-distress-2017-12-05

In short Beneish m score is a model that measures financial statement manipulation while john Campbell's financial distress model measures the probability of financial distress. But the articles go in more detail on their calculations. It would be really awesome if these models were in the fintel stock screener.
